source/19-Psalms.usfm.db

```diff
@@ -5717,7 +5717,7 @@
 \v 36 “that his line should endure forever,
 \q2 and his throne as the sun before me,
 \q
-\v 37 firm as the moon which for ever
+\v 37 firm as the moon which [us:forever|cth:for ever]
 \q2 and ever is fixed in the sky.” \qs Selah\qs*
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".